index.twig

```diff
@@ -135,6 +138,18 @@
             </div>
           </div>
         {% endfor %}
+        {% if section.markup and options.showMarkup %}
+          <details class="kss-markup kss-style">
+            <summary>
+              {% if section.markupFile %}
+                Markup: <code>{{ section.markupFile }}</code>
```

Comment:
We're including this above the example already, probably doesn't need to be repeated